openfisca_us/variables/household/demographic/geographic/zip_code.py

```diff
@@ -0,0 +1,8 @@
+from openfisca_us.model_api import *
+
+
+class zip_code(Variable):
+    value_type = str
+    entity = Household
+    label = "Zip code"
```

Comment:
Acronym for Zone Improvement Plan
```suggestion
    label = "ZIP code"
```